apis/r/R/RcppExports.R

```diff
@@ -96,22 +94,16 @@ shape <- function(uri, config = NULL) {
 #' returns a boolean, and \code{sr_next} returns an Arrow array helper object.
 #'
 #' @examples
-#' \dontrun{
+#' uri <- extract_dataset("soma-dataframe-pbmc3k-processed-obs")
 #' ctx <- tiledb::tiledb_ctx()
-#' uri <- "test/soco/pbmc3k_processed/obs"
-#' sr <- sr_setup(uri, config=as.character(tiledb::config(ctx)), loglevel="auto")
+#' sr <- sr_setup(uri, config=as.character(tiledb::config(ctx)))
 #' rl <- data.frame()
 #' while (!sr_complete(sr)) {
-#'     sr |>
-#'         sr_next() |>
-#'         as_arrow_table() |>
-#'         collect() |>
-#'         as.data.frame() |>
-#'         data.table() -> D
-#'     rl <- rbind(rl, D)
+#'   dat <- sr_next(sr)
+#'   rb <- arrow::RecordBatch$import_from_c(dat$array_data, dat$schema)
+#'   rl <- rbind(rl, as.data.frame(rb))
```

Comment:
We could also rbind with the as.data.frame.  I think we do that in another spot.  Works either way.

Reply:
I think it's already using rbind with as.data.frame:

```
rbind(rl, as.data.frame(rb))
```

did you mean _without_ as.data.frame? Or am I misunderstanding?

Reply:
Outside of this example we don't really deal with the pair of pointers any more.  Maybe

```r
at <- soma_array_to_arrow_table( sr_next( sr ) )
```

is better here too?